felix/bpf/conntrack/v4/map.go

```diff
@@ -206,22 +206,22 @@ const (
 	TypeNATForward
 	TypeNATReverse
 
-	FlagNATOut          uint16 = (1 << 0)
-	FlagNATFwdDsr       uint16 = (1 << 1)
-	FlagNATNPFwd        uint16 = (1 << 2)
-	FlagSkipFIB         uint16 = (1 << 3)
-	FlagReserved4       uint16 = (1 << 4)
-	FlagReserved5       uint16 = (1 << 5)
-	FlagExtLocal        uint16 = (1 << 6)
-	FlagViaNATIf        uint16 = (1 << 7)
-	FlagSrcDstBA        uint16 = (1 << 8)
-	FlagHostPSNAT       uint16 = (1 << 9)
-	FlagSvcSelf         uint16 = (1 << 10)
-	FlagNPLoop          uint16 = (1 << 11)
-	FlagNPRemote        uint16 = (1 << 12)
-	FlagNoDSR           uint16 = (1 << 13)
-	FlagNoRedirPeer     uint16 = (1 << 14)
-	FlagClusterExternal uint16 = (1 << 15)
+	FlagNATOut      uint16 = (1 << 0)
+	FlagNATFwdDsr   uint16 = (1 << 1)
+	FlagNATNPFwd    uint16 = (1 << 2)
+	FlagSkipFIB     uint16 = (1 << 3)
+	FlagReserved4   uint16 = (1 << 4)
+	FlagReserved5   uint16 = (1 << 5)
+	FlagExtLocal    uint16 = (1 << 6)
+	FlagViaNATIf    uint16 = (1 << 7)
+	FlagSrcDstBA    uint16 = (1 << 8)
+	FlagHostPSNAT   uint16 = (1 << 9)
+	FlagSvcSelf     uint16 = (1 << 10)
+	FlagNPLoop      uint16 = (1 << 11)
+	FlagNPRemote    uint16 = (1 << 12)
+	FlagNoDSR       uint16 = (1 << 13)
+	FlagNoRedirPeer uint16 = (1 << 14)
+	FlagSetDSCP     uint16 = (1 << 15)
```

Comment:
Is the String() missing the stringification of this flag? We missed a few lately

Reply:
It does! I never add it before. It's added now. Thanks for the tip.